docs_src/microservices/core/data/Ch-CoreData.md

```diff
@@ -1,102 +1,170 @@
-gateway) until the data can be moved "north" and then exported to
-Enterprise and cloud systems.
-
-Other services, such as a Scheduling services, within EdgeX Foundry and
-potentially outside of EdgeX Foundry, access the device and sensor data
-stored on the gateway only through the Core Data service. Core Data
-provides a degree of security and protection of the data collected by
-devices and sensors while the data is at the edge.
-
-Core Data uses a REST API for moving data into and out of the local
-storage. In the future, the microservice could be expandable to allow
-data to be accessed via other protocols such as MQTT, AMQP, and so
-forth. Core Data moves data to the Export Service layer via ZeroMQ by
-default. An alternate configuration of the Core Data microservice allows
-the data to be distributed to the Export Services via MQTT, but would
-also require the installation of a broker such as ActiveMQ.
-
-The Rules Engine microservice receives its data from the Export
-Distribution microservice by default. Where latency or volume are of
-concern, an alternate configuration of the Rules Engine microservices
-allows it to also get its data directly from Core Data via ZeroMQ (it
-becomes a second subscriber to the same Export Services ZeroMQ
-distribution channel).
+The core data micro service provides centralized persistence for data collected by [devices](../../../general/Definitions.md#device). 
+Device services that collect sensor data call on the core data service to
+store the sensor data on the edge system (such as in a
+[gateway](../../../general/Definitions.md#gateway)) until the data gets moved "north" and then exported to
+Enterprise and cloud systems.  Core data persists the data in a local database.  [Redis](https://redis.io/) is used by default, but a database abstraction layer allows for other databases to be used.
+
+!!! Note
+        EdgeX has used [MongoDB](https://www.mongodb.com/) in the past. Through the Geneva release, MongoDB is still supported but is considered deprecated.  Alternate (both open source and commercial) implementations have also been provided in the past. 
+
+Other services and systems, both within EdgeX Foundry and
+outside of EdgeX Foundry, access the sensor data
+only through the core data service. Core data
+could also provide a degree of security and protection of the data collected while the data is at the edge.
+
+Core data has a REST API for moving data into and out of the local
+storage. In the future, core data could be expandable to send or access sensor data via other protocols such as MQTT, AMQP, etc. Core data moves data to the application service (and [edge analytcs](../../../general/Definitions.md#edge-analytics)) via ZeroMQ by default. 
+A message bus abstraction allows core data to use an alternate message bus technology, like MQTT, to send data to the application services.  Use of MQTT requires the installation of a broker such as ActiveMQ.
```

Comment:
The way this is stated "A message bus abstraction allows core data to use an alternate message bus technology..." kind of sounds like you need to add the abstraction yourself and need to do the work. Instead of clearly saying that EdgeX has provided a message bus abstraction that supports ZeroMQ by default, and MQTT (with the addition of an MQTT Broker) and you can add your own implementation as needed.

Reply:
reworded - largely around your wording.